content/en/docs/reference/setup-tools/kubeadm/kubeadm-init.md

```diff
@@ -438,7 +439,14 @@ Here `v1.8.x` means the "latest patch release of the v1.8 branch".
 
 `${ARCH}` can be one of: `amd64`, `arm`, `arm64`, `ppc64le` or `s390x`.
 
-If using `--feature-gates=CoreDNS=true` image `coredns/coredns:1.0.2` is required (instead of the three `k8s-dns-*` images).
+In Kubernetes versions before 1.11 and if using `--feature-gates=CoreDNS=true`,
+the image `coredns/coredns:1.0.2` is required (instead of the three `k8s-dns-*` images).
+
```

Comment:
Thought I edited this once, but don't see it in the review history. I'll check for dupe content in different files about kubeadm. In the meantime, though, change ll442-443 to:

"If you run Kubernetes version 1.10 or earlier, and if you set `--feature-gates=CoreDNS=true`, you must also use the image `coredns/coredns:1.0.2`, instead of the three `k8s-dns-*` images."